### ISSUE TITLE
Fruit fly mitochondria

### ISSUE THREAD

**Issue by wangjiawen2013** [refgenie/refgenie#238]
Hi,
I can see human and mouse mitochondiral chromosome (rCRSd and mouse_chrM2x ), while cannot find that of fruit fly. Can I just extract it from drosophila melanogaster genome and build myself ?

**Comment by nsheff** [comment]
Yes, you can. Instructions are here: http://refgenie.databio.org/en/latest/build/

just do get a fasta file with the mtdna you want and do:

```
refgenie build dm_mtDNA/fasta --files refgene=dm_mtDNA.fa.gz
```
